app/javascript/admin/controllers/config_controller.js

```diff
@@ -139,14 +139,29 @@ export default class ConfigController extends Controller {
     const previewLogo = document.querySelector(
       '#logo-upload-preview .site-logo__img',
     );
+    const communityName = document.querySelector(
+      '.site-logo .site-logo__community-name',
+    );
 
     if (!previewLogo) {
       return;
     }
 
-    for (const logo of document.querySelectorAll('.site-logo__img')) {
-      if (logo !== previewLogo) {
-        logo.src = previewLogo.src;
+    // if we are showing the community name because this is the first time that the
+    // creator is uploading a logo, then we want it to replace the community name
+    if (communityName) {
```

Comment:
We changed the way that `_logo_html.erb` renders. Essentially if the feature flag is enabled and there is no resized_logo, it will show the community_name. This will prepare us to remove `logo_svg` completely. 

Hence, in this code we needed to cater for a creator that may have only his community name set because he has not uploaded a logo before.